app/components/TableSearchFieldSelector.tsx

```diff
@@ -9,7 +9,7 @@ export default function TableSearchFieldSelector({
   // todo: bug here where changing the search field clears the search input if it hasn't been searched
```

Comment:
is the bug still there? maybe i dont understand how to get the bug to happen, but i couldnt reproduce

Reply:
looks like it's fixed! 🙏